pyproject.toml

```diff
@@ -0,0 +1,24 @@
+[tool.flake8]
+# TODO add python directory here in later PR
+filename = ["./elements/**/*.py"]
+ignore = ["E501"]
+# Black compatibility
+max-line-length = 88
+extend-ignore = ["E203", "W503"]
+
+[tool.black]
+exclude = '^\/tools\/.*\.py$|^\/exampleCourse\/.*\.py$|^\/testCourse\/.*\.py$|^\/node_modules\/.*\.pyi?$'
```

Comment:
IMO it'd be great to include all of these (except `node_modules`) in the future. Can we do that as a relatively quick followup?

Reply:
Sure! The only thing that has to stay out is some of the tools, there's some Python 2 in there.